daemons/controld/controld_remote_ra.c

```diff
@@ -1224,6 +1233,21 @@ remote_ra_fail(const char *node_name)
     "[@" XML_LRM_ATTR_TASK "='stonith']/" XML_GRAPH_TAG_DOWNED \
     "/" XML_CIB_TAG_NODE
 
+/* A stop action on a remote node connection resource looks like:
+ *
+ *   <pseudo_event id="86" operation="stop" operation_key="remote01_stop_0">
+ *      <attributes CRM_meta_expire_attrs="true" CRM_meta_name="stop"
+ *                  CRM_meta_timeout="60000" crm_feature_set="3.17.4"
+ *                  server="rhel9-cluster-3"/>
+ *   </pseudo_event>
+ *
+ * The following xpath query returns the pseudo_event node that
+ * has an attributes sub-element with CRM_meta_expire_attrs="true".
+ */
+#define XPATH_PSEUDO_EXPIRE "//" XML_GRAPH_TAG_PSEUDO_EVENT \
```

Comment:
btw if we know the complete path, it's better to specify than use "//" since the search will be much more efficient

Reply:
Yeah, it looks like the top-level XML node this function gets is pseudo_event.  Note that I patterned this on XPATH_PSEUDO_FENCE above, which also uses "//".  I'll add a commit to fix that one too.